src/server/auth/cmds/cmds_test.go

```diff
@@ -192,25 +193,6 @@ func TestConfig(t *testing.T) {
 		`).Run())
 }
 
-// TestGetAuthTokenNoSubject tests that 'pachctl get-auth-token' infers the
-// subject from the currently logged-in user if none is specified on the command
-// line
-func TestGetAuthTokenNoSubject(t *testing.T) {
```

Comment:
Just checking: it looks like support for this is getting dropped? I added a note to `GetAuthTokenInTransaction` too.

Reply:
Yup, dropping support for this  - it was easier to do it here and simplify the permissions checks but I think this API will also change more before 2.0.